src/tools/auth0/client.ts

```diff
@@ -167,7 +167,7 @@ export default function pagedClient(client: BaseAuth0APIClient): Auth0APIClient
       frequencyLimit: API_FREQUENCY_PER_SECOND,
       frequencyWindow: 1000, // 1 sec
     }),
-  };
+  } as unknown as Auth0APIClient;
```

Comment:
Looks very weird. Can the `unknown` be removed or this type cast otherwise cleaned up?

Reply:
**Update:** Looking at this with Frederik again, we should probably find a way to exclude the pool property or facilitate pagination some other way that can retain the `Auth0APIClient` type.